### ISSUE TITLE
Cannot insert button blocks

### ISSUE THREAD

**Issue by draganescu** [WordPress/gutenberg#16959]
**Describe the bug**
Adding a button block in blocks using inner blocks fails with a console message: `TypeError: Cannot read property 'align' of undefined`.

**To reproduce**
1. Add a cover block
2. Pick an image
3. Replace the paragraph block added after step 2 with a button using the / inserter.
4. Observe the bug

or:
 
1. Add a media+text block
2. Replace the paragraph in the text section with a button using the / inserter.
3. Observe the bug

**Expected behavior**
The button to be added.

**Comment by youknowriad** [comment]
Let's close this as a duplicate of #17013
